ngraph/core/include/ngraph/op/util/evaluation_context.hpp

```diff
@@ -0,0 +1,132 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <ngraph/output_vector.hpp>
+#include <ngraph/variant.hpp>
+#include "ngraph/op/util/variable.hpp"
+#include "ngraph/op/util/variable_value.hpp"
+
+namespace ngraph
+{
+    class Variable;
```

Comment:
Why do you use forward declaration? Can we just include the right header?

Reply:
My preference is to use forward declarations in header files where possible. At least it reduces coupling and compilation time.
But here there is already `#include <...variable.hpp>`, so either include or forward declaration is not needed here

Reply:
fixed